gcp_variant_transforms/transforms/variant_to_bigquery.py

```
@@ -147,3 +159,79 @@ def expand(self, pcoll):
                       beam.io.BigQueryDisposition.WRITE_APPEND
                       if self._append
                       else beam.io.BigQueryDisposition.WRITE_TRUNCATE))))
+
+  def _update_bigquery_schema(self, update_schema):
+    # type: (bool) -> None
+    if not update_schema or not self._schema:
+      return
+    # if table does not exist, do not need to update the schema.
+    output_table_re_match = re.match(
+        r'^((?P<project>.+):)(?P<dataset>\w+)\.(?P<table>[\w\$]+)$',
+        self._output_table)
+    credentials = GoogleCredentials.get_application_default().create_scoped(
+        ['https://www.googleapis.com/auth/bigquery'])
+    client = bigquery.BigqueryV2(credentials=credentials)
+    try:
+      project_id = output_table_re_match.group('project')
+      dataset_id = output_table_re_match.group('dataset')
+      table_id = output_table_re_match.group('table')
+      found_table = client.tables.Get(bigquery.BigqueryTablesGetRequest(
+          projectId=project_id,
+          datasetId=dataset_id,
+          tableId=table_id))
+    except exceptions.HttpError:
+      return
+
+    new_schema = bigquery.TableSchema()
+    new_schema.fields = _merge_field_schemas(self._schema.fields,
+                                             found_table.schema.fields)
+    original_schema = found_table.schema
+    found_table.schema = new_schema
+    try:
+      client.tables.Update(bigquery.BigqueryTablesUpdateRequest(
+          projectId=project_id,
+          datasetId=dataset_id,
+          table=found_table,
+          tableId=table_id))
+    except exceptions.HttpError:
+      found_table.schema = original_schema
```

I think it's better to raise the error here and fail. If someone has explicitly set update_schema and we fail for any reason, we don't want to even start the pipeline.
You can do something like: `raise RuntimeError('BigQuery schema update failed: %s' % str(e))`